UNIXFS.md

```diff
@@ -59,6 +63,76 @@ For files that are comprised of more than a single block, the 'Type' field will
 
 This data is serialized and placed inside the 'Data' field of the outer merkledag protobuf, which also contains the actual links to the child nodes of this object.
 
+## Metadata
+
+UnixFS currently supports three metadata fields:
+
+* `mimeType` -- The mime-type of the file. This generally shouldn't be used.
```

Comment:
perhaps this "shouldn't" could be explained? is it deprecated? does it have a special case use?

Reply:
I'm also very confused by this.  I didn't know it exists in any existing data; it won't exist in Unixfsv2; and I thought we'd had multiple roundabouts in github issues in which multiple people highlighted the issue that mimetypes are not generally considered a property of a filesystem (rather, they tend to show up in applications, servers, and other things that are routing control flow in some way) and that they shouldn't thus be in unixfsv{n}.

Reply:
This type was already reserved by the _existing_ metadata feature.

Reply:
Is there any reason not to leave it there, in the deprecated state that it becomes there?

If it's deprecated, it can still be used -- but "shouldn't" be.  Which is what this text says anyway.  If it's deprecated and we don't want it to be used, we should prefer to draw proportionally less attention to it, and leaving it in the depricated notes should be sufficient for that.